addOns/oast/CHANGELOG.md

```diff
@@ -6,7 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
-
+### Changed
+- Updated service URLs for Boast and Interactsh in help content.
+- Updated service configuration details in the Interactsh dialog and payload script template.
```

Comment:
e.g. "No longer provide a default server URL for Interactsh due to (random) unavailability". I'd also argue that the script should be left empty for the same reason, if not it should be added as other entry in the changelog.